app/test_main.py

```diff
@@ -1,3 +1,29 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_output",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 28, [2, 2]),
+        (28, 29, [3, 3])
```

Comment:
You should include edge cases as per the checklist. Consider adding tests for edge situations such as negative numbers, zero, or very large numbers to ensure the function handles these cases correctly.